hydra-web/src/js/modules/alias.js

```diff
@@ -131,7 +131,7 @@ function(
                     "bVisible":true,
                     "bSearchable":true,
                     "mRender":function(val,type,data){
-                        return val.join(",");
+                        return "<a href='#jobs/"+encodeURIComponent(val)+"/conf'>"+val+"</a>";
```

Comment:
If there are multiple job ids you can see one long string which is appending ids, I've searched an alias in spawn prod., but couldn't find one which has multiple ids. But if there is a case where using multiple ids, returning a list of ids which can click seperate ids might be one of possible approaches. BTW, I'm wondering the scenario when it's using muptile id for an alias.